api/v1alpha1/virtualbackend_types.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	// KindSecurityPolicy is the name of the SecurityPolicy kind.
+	KindVirtualBackend = "VirtualBackend"
+)
+
+type Body string
```

Comment:
Maybe the body should be a `[]byte` instead of a string? Just like in `Secret` resources. That would allow having arbitrarily complex direct responses, including images, and would also clearly separate the Envoy Gateway CRD from the content of the direct response.